frontend/src/cards/ui/SviAlert.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+import variables from "../../styles/variables.module.scss";
+import { MetricQueryResponse } from "../../data/query/MetricQuery";
+import { Fips } from "../../data/utils/Fips";
+
+interface SviAlertProps {
+  svi: number;
+  sviQueryResponse: MetricQueryResponse;
+  fips: Fips;
+}
+
+const findRating = (svi: number) => {
+  if (svi < 0.34) {
+    return "low";
+  }
+  if (svi > 0.67) {
+    return "high";
+  } else return "medium";
+};
+
+const findColor = (rating: string) => {
+  if (rating === "high") {
+    return "#d32f2f";
```

Comment:
If possible I'd love to stay consistent with our styling. Specifically:
- make a scss module for this new SviAlert component
- import scss variables
- only use sass variables like `sass.altGreen`; add new entries to the variable files if we don't have an existing color that works